src/modules/twap/pure/ConfirmDetailsItem/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from 'react'
+
+import { CornerDownRight } from 'react-feather'
+
+import { RowFixed } from 'legacy/components/Row'
+import { MouseoverTooltipContent } from 'legacy/components/Tooltip'
+
+import { TextWrapper } from 'modules/swap/pure/Row/styled'
+import { StyledInfoIcon } from 'modules/swap/pure/styled'
```

Comment:
should these be exported from the index if needed outside? Or these are too generic and should be moved to a even more common module?